### ISSUE TITLE
Plaid::exchangeToken() POST URL

### ISSUE THREAD

**Issue by acirinelli** [travoltron/plaid#10]
It appears the POST URL to exchange a public_token for an access_token has been changed in the API from **/exchange_token** to **/item/public_token/exchange**

https://plaid.com/docs/api/#exchange-token-flow

**Comment by travoltron** [comment]
Yeah, this package was built before Plaid resurfaced the API. Rest assured,
there's a fix coming soon.

**Comment by acirinelli** [comment]
Sorry, I just realized you already had an issue going for this: https://github.com/travoltron/plaid/issues/9

Curious when you think the new package will be ready? I see you've made some commits the past few days, sweet! For now I've just made a copy of your package into a custom dir and patched it together so I can keep working. There's another Plaid package available, but it doesn't seem to be as good as yours and not at all documented.

**Comment by travoltron** [comment]
I'm hoping within about 3 weeks. Thanks for the kind words.

**Comment by designloud** [comment]
@travoltron running into the same issue here.  First time with your package and using Plaid so Im anxious to see if this is going to work well for my application

**Comment by acirinelli** [comment]
@designloud I used this work around and modified the package to fit my needs: https://stackoverflow.com/questions/31080696/safely-edit-a-third-party-composer-vendor-package-in-laravel-prevent-losing
